styles/app.css

```diff
@@ -0,0 +1,115 @@
+body {
+  margin: 0;
+  font-family: sans-serif;
+}
+
+ul {
+  padding-left: 0;
+  list-style: none;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+h3 {
+  font-weight: normal;
+}
+
+.container {
+  padding: 6rem;
+}
+
+.top-bar {
+  background: #66b5e8;
+  overflow: auto;
+  color: #fefefe;
+}
+
+.top-bar__title {
+  text-align: left;
```

Comment:
Try using float instead of text align which should allow content to line up instead of being on separate lines here.